delfin/drivers/utils/ssh_client.py

```diff
@@ -241,15 +242,17 @@ def put(self, conn):
         super(SSHPool, self).put(conn)
 
     def do_exec(self, command_str):
-        result = None
+        result = ''
+        eventlet.monkey_patch()
```

Comment:
Why add `monkey_patch()` here, we do this at the entry of every process.

Reply:
18/5000 
This module needs to set this place before setting the timeout

Reply:
What would happend if we not call `eventlet.monkey_patch()` here

Reply:
This is the place to mark this part of the code as a green thread. You don't have to wait for the execution to complete. If the set time exceeds, you can proceed directly.